rpcs3/CMakeLists.txt

```diff
@@ -129,7 +129,7 @@ if(NOT MSVC)
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wl,--allow-multiple-definition")
 	endif()
 
-	add_compile_options(-msse -msse2 -mcx16)
+	add_compile_options(-msse -msse2 -mcx16 -mrtm)
```

Comment:
Very wrong, it assumes compilation for processors only with rtm support.

Reply:
How would you solve the undeclared indentifier knowing the file `immintrin.h` contains this?
```
#ifdef __RTM__
    #include <rtmintrin.h>
#endif
```
Source: https://web.archive.org/web/20150502153249/http://clang.llvm.org:80/doxygen/immintrin_8h_source.html

Reply:
It's just a constant, defined by x86 ISA

Reply:
So, what would be your suggestion to fix this issue without using `-mrtm`?
- Add `#define __RTM__ 1` in the code?
- Add `#ifndef _XABORT_RETRY #define _XABORT_RETRY	 (1 << 1) #endif` at the top of the JIT.cpp?
- Bump the minimal version of clang required?
- Something else?

Reply:
Second is fine, first is wrong too.